apps/studio/src/migration/20210811_add_sort_to_settings.js

```diff
@@ -0,0 +1,16 @@
+export default {
+  name: "20210811_add_sort_to_settings",
+  async run(runner) {
+    const queries = [
+      `
+        INSERT INTO user_setting (key, defaultValue, valueType)
+        VALUES
+        ('sortOrder', 'id', '0')
```

Comment:
could we call this `connectionSortOrder` rather than `sortOrder`? That way we can have a different sort order for other stuff.